<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreFirstWindow</class>
 <widget class="QDialog" name="PreFirstWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="nameLineEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Input your project's name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="directoryLineEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose your save directory</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cmakeLineEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Input the way to cmake</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="compilerLineEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Input the way to compiler</string>
   </property>
  </widget>
  <widget class="QToolButton" name="directoryButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>80</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QToolButton" name="cmakeButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>120</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QToolButton" name="compilerButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>160</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okPushBut">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>250</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
